api/file.go

```diff
@@ -94,28 +89,23 @@ func uploadFile(c *Context, w http.ResponseWriter, r *http.Request) {
 		buf := bytes.NewBuffer(nil)
 		io.Copy(buf, file)
 
-		ext := filepath.Ext(files[i].Filename)
-
 		uid := model.NewId()
 
 		path := "teams/" + c.Session.TeamId + "/channels/" + channelId + "/users/" + c.Session.UserId + "/" + uid + "/" + files[i].Filename
```

Comment:
Good catch, I'll add sanitization to prevent relative paths and add a unit test for it